src/App.js

```diff
@@ -1,12 +1,16 @@
 import React from 'react';
 import {SafeAreaProvider} from 'react-native-safe-area-context';
+
 import CustomStatusBar from './components/CustomStatusBar';
 import Expensify from './Expensify';
+import ErrorBoundary from './components/ErrorBoundary';
 
 const App = () => (
     <SafeAreaProvider>
         <CustomStatusBar />
-        <Expensify />
+        <ErrorBoundary errorMessage="[App] A Crash was intercepted by the main error boundary">
```

Comment:
Can we update the errorMessage to be `E.cash crash caught by error boundary`? We can also remove the `[App]` from the message as well.